app/assets/javascripts/filter_set.js.coffee

```diff
@@ -0,0 +1,27 @@
+class @FilterSet
+  constructor: (@$root, @key, @val) ->
+
+  $: (selector) ->
+    @$root.find(selector)
+
+  children: ->
+    @$("[data-filter-key=#{ @key }][data-filter-value=#{ @val }]")
+
+  adjacentChildren: ->
```

Comment:
> Siblings (brothers or sisters) are nodes with the same parent

http://www.w3schools.com/js/js_htmldom_navigation.asp

This method could be named `childrenOfSiblingsOfParent`, but that doesn't seem much clearer :confused: Thoughts?